src/code/components/basket-set.js

```diff
@@ -0,0 +1,64 @@
+import React, {PropTypes} from 'react';
+
+/**
+ * @param {number} rows - Option number of rows. If defined, it will be fixed at that. Otherwise, it
+ *                        will default to 1 when there are no orgs, and grows as more rows are needed.
+ * @param {number} tightenRows - If given, will shrink the vertical height of the pen by this amount
```

Comment:
Do you need to specify this manually, or could it be derived from inspecting the overall screen size and content bounds?